swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -2238,6 +2244,7 @@ paths:
         - $ref: "#/components/parameters/PSU-Corporate-ID"
         - $ref: "#/components/parameters/PSU-Corporate-ID-Type"
         - $ref: "#/components/parameters/TPP-Redirect-Preferred"
+        - $ref: "#/components/parameters/TPP-Decoupled-Preferred"
```

Comment:
Relevant Change: TPP-Decoupled-Preferred is new
Adoption approach: Follow the change